guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,326 @@
+- Bitcoin.com mobile wallet
+
+***
+
+## Manual backup / Recovery-phrase
+
+Manual backup of the recovery phrase have been the most common private key management scheme by far, since its proposal with [BIP39]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki'}}) in 2013. If you have used any non-custodial bitcoin product you are likely to have experienced the onboarding requirements of manual backups. 
+
+After creating a new wallet, you will be asked to manually backup a 12 (or 24 in rare instances) word recovery-phrase to a *safe place*. As the next step it will often ask you to verify that you did save it by having you input the phrase in the correct order. 
+
+This scheme is suitable for target-audiences who are already familiar with bitcoin and procedures for secure off-line backups of the recovery-phrase. It is not suited for complete beginners. When told to store the backup in a safe way off-line, bitcoin-beginners in reality often take a screenshot, write it down in plain text somewhere on their mobile device, computer or a piece of paper on the fridge, or simply don’t back it up at all. This has the risk of achieving the opposite of what we want; high risk of self-inflicted loss and low to medium security in terms of third party theft.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/recovery-phrase.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/recovery-phrase@2x.jpg"
+   alt-text = "Recovery-phrase illustration"
+   caption = "Recovery-phrases should be stored off-line on metal or paper backups."
+   width = 800
+   height = 400
+%}
+
+#### Pros 
+- Manual backups done well can provide very high security
+- Good interoperability
+
+#### Cons 
+- Requires significant effort from users to achieve safe and redundant backups
+- High onboarding friction
+
+### How it works
+The wallet-application will generate a 12 or 24 word phrase (seed / recovery phrase) from which all the wallet’s keys can be derived. This means the user can have access to the wallet from any compatible wallet-application with the phrase, even if they lose the device or software.
+
+This can be an effective way to reduce the risk of loss from theft if the backup is off-line in a safe place, but puts more of the op-sec burden on the individual user. The security and risk will only be as good as how they backup the recovery phrase. To get the best security out of recovery phrases they should be in several redundant and secure places off-line. You also need to take resistance to physical theft, fire and other destructive events into account. A fair amount of effort and knowledge is required for this. 
+
+A full guide to manual backup best practices for users is out of scope for this chapter but the [Do's and Dont's of Bitcoin key management]({{ https://blog.keys.casa/the-dos-and-donts-of-bitcoin-key-management/ }}) is a good starting point.
```

Comment:
This link doesn't work locally, redirects to this page.

Reply:
Confirmed working in the preview build linked from GitHub